integration_tests/mockApis/cemo.ts

```diff
@@ -125,6 +125,81 @@ const getOrder = (options: GetOrderStubOptions = defaultGetOrderOptions): SuperA
     },
   })
 
+type CreateOrderStubOptions = {
+  httpStatus: number
+  id?: string
+  status?: string
+}
+
+const defaultCreateOrderOptions = {
+  httpStatus: 200,
+  id: uuidv4(),
+  status: 'IN_PROGRESS',
+}
+
+const createOrder = (options: CreateOrderStubOptions = defaultCreateOrderOptions): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/cemo/api/CreateForm\\?title=(.*)`,
+    },
+    response: {
+      status: options.httpStatus,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody:
+        options.httpStatus === 200
+          ? {
+              id: options.id,
+              status: options.status,
+              deviceWearer: {
+                nomisId: null,
+                pncId: null,
+                deliusId: null,
+                prisonNumber: null,
+                firstName: null,
+                lastName: null,
+                alias: null,
+                dateOfBirth: null,
+                adultAtTimeOfInstallation: null,
+                sex: null,
+                gender: null,
+                disabilities: null,
+              },
+              deviceWearerContactDetails: {
+                contactNumber: null,
+              },
+              additionalDocuments: [],
+            }
+          : null,
+    },
+  })
+
+type PostOrderStubOptions = {
+  httpStatus: number
+  id?: string
+  status?: string
+}
+
+const defaultPostOrderOptions = {
+  httpStatus: 200,
+  id: uuidv4(),
+  status: 'IN_PROGRESS',
+}
+
+const postOrder = (options: PostOrderStubOptions = defaultPostOrderOptions): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPattern: `/cemo/api/order/${options.id}/device-wearer`,
+    },
+    response: {
+      status: options.httpStatus,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      body: options.httpStatus === 200 ? '{{{request.body}}}' : null,
+      transformers: ['response-template'],
+    },
+  })
+
```

Comment:
I think the naming may be wrong, this endpoint isn't posting the entirety of the order, just the device-wearer

Reply:
This is renamed more appropriately now